app/src/main/res/values/styles.xml

```diff
@@ -154,7 +154,7 @@
         <item name="android:paddingEnd">@dimen/default_margin</item>
         <item name="android:textSize">14sp</item>
         <item name="android:fontFamily">sans-serif</item>
-        <item name="android:textColor">@color/text_emphasis_low</item>
+        <item name="android:textColor">@color/text_emphasis_medium</item>
```

Comment:
I checked the the design for  the other usage of HelpText and it also says text_emphasis_medium. So this change fixes both usages.